### PR TITLE
Update roster file

### PR DESCRIPTION
**Pull request by Jayesh-sathe**

### CODE REVIEW

**Comment by kbuffardi** [comment]
Make sure to resolve the conflicts

**Comment by Jayesh-sathe** [comment]
Added Jayesh to roster

**Comment by Jayesh-sathe** [comment]
Added Jayesh in roster

File: roster.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 * Kevin Buffardi (instructor)
 * Devam Sharma (student)
 * Gopal Raigoni (student)
+* Jayesh Sathe (student)
+* Shelley Wong (student)
```
Comment (kbuffardi) [inline roster.md]: There's a duplicate here, where you have Shelley (incorrectly) before Padam, but also (correctly) after (line 9)

Comment (Jayesh-sathe) [reply roster.md]: Hi Kevin,
I have updated the names alphabetically. Please check.
Thank You.

**Comment by Jayesh-sathe** [comment]
Updating names alphabetically and adding Jayesh

**Comment by kbuffardi** [comment]
The names seem to be in the right order now, but there's a conflict that needs to be resolved

**Comment by Jayesh-sathe** [comment]
Resolving the conflict and adding Jayesh

**Pull request merged**